gustaf/utils/tictoc.py

```diff
@@ -119,6 +119,6 @@ def summary(self, log=True, print_=False):
             if log:
                 self._logger(*message)
             if print_:
-                print(*message)
+                print(*message)  # noqa: T201
```

Comment:
print statements are forbidden following rule T201. Here we can ignore this rule due to explicitly asking for a print of the message.